OrbitGl/TimeGraph.cpp

```diff
@@ -763,10 +763,12 @@ void TimeGraph::DrawTracks(GlCanvas* canvas, PickingMode picking_mode) {
         std::string process_name = GOrbitApp->GetCaptureData().process_name();
         thread_track->SetName(process_name);
         thread_track->SetLabel(process_name + " (all threads)");
+        thread_track->SetNumberOfPrioritizedTrailingCharacters(strlen("(all threads)"));
```

Comment:
Not sure why this is done on every draw, but it seems to be the pattern here. We should probably clean that up in another PR.